pack/tde/sfk_encounter.json

```diff
@@ -0,0 +1,873 @@
+		"health": 4,
+		"illustrator": "Stephen Somers",
+		"name": "Tenebrous Nightgaunt",
+		"pack_code": "sfk",
+		"position": 150,
+		"quantity": 2,
+		"text": "Hunter.\n<b>Forced</b> - When Tenebrous Nightgaunt leaves play as a result of an act's setup: Set it aside (it keeps all tokens and attachments). After resolving the act's setup, put it back into play at any [[City]] location.",
+		"traits": "Monster. Nightgaunt.",
+		"type_code": "enemy"
+	},
+	{
+		"code": "06151",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 36,
+		"enemy_damage": 1,
+		"enemy_evade": 1,
+		"enemy_fight": 1,
+		"enemy_horror": 1,
+		"faction_code": "mythos",
+		"health": 2,
+		"illustrator": "Matthew Cowdery",
+		"name": "Pack of Vooniths",
+		"pack_code": "sfk",
+		"position": 151,
+		"quantity": 2,
+		"text": "Swarming 1.\nWhile the host Pack of Vooniths has no attached swarm cards, it gets +2 fight and +2 evade.",
+		"traits": "Creature. Monster.",
+		"type_code": "enemy"
+	},
+	{
+		"code": "06152",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 38,
+		"enemy_evade": 5,
+		"enemy_fight": 2,
+		"enemy_horror": 1,
+		"faction_code": "mythos",
+		"health": 2,
+		"illustrator": "Matthew Cowdery",
+		"name": "Nightriders",
+		"pack_code": "sfk",
+		"position": 152,
+		"quantity": 2,
+		"text": "Swarming 1.\nWhile the host Nightriders has no attached swarm cards, it gets -3 evade.\n<b>Forced</b> - After you evade Nightriders: Discard it.",
+		"traits": "Creature. Monster.",
+		"type_code": "enemy"
+	},
+	{
+		"code": "06153",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 40,
+		"faction_code": "mythos",
+		"illustrator": "Ian Kirkpatrick",
+		"name": "Song of the Magah Bird",
+		"pack_code": "sfk",
+		"position": 153,
+		"quantity": 2,
+		"text": "<b>Revelation</b> - Attach to your location.\n<b>Forced</b> - After you move out of attached location: Take 1 horror, place 1 doom on the current agenda, and discard Song of the Magah Bird. This effect can cause the current agenda to advance.\n[action]: Test [willpower] (4) to resist the call or [combat] (4) to drive away the birds. If you succeed, discard Song of the Magah Bird.",
+		"traits": "Curse.",
+		"type_code": "treachery"
+	},
+	{
+		"code": "06154",
+		"encounter_code": "the_search_for_kadath",
+		"encounter_position": 42,
+		"faction_code": "mythos",
+		"illustrator": "Leanna Crossan",
+		"name": "Wondrous Lands",
+		"pack_code": "sfk",
+		"position": 154,
+		"quantity": 2,
+		"text": "<b>Revelation</b> - If there are no clues on your location, discard Wondrous Lands, and it gains surge. Otherwise, attach Wondrous Lands to your location.\nAttached location gets -2 shroud.\n<b>Forced</b> - After you successfully investigate attached location: Take 1 horror, place 1 doom on the current agenda, and discard Wondrous Lands. This effect can cause the current agenda to advance.",
+		"traits": "Power.",
+		"type_code": "treachery"
+	}
+]
```

Comment:
Great effort - really respect how tidy these are, and what a great job you've done! 

Waiting to hear from discord about encounter_position and encounter_code for the b sides of cards, but otherwise once you've done these, all good.

Reply:
Thanks :-)